src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,64 @@
+import { useState, useEffect } from "react";
+import "./home.css";
+import { BannerCard, ProductCard } from "../../components";
+import keyboardImg from "../../assets/banner-images/banner-keyboard.png";
+import xboxImg from "../../assets/banner-images/xbox-controller.png";
+import vrImg from "../../assets/banner-images/vr.png";
+import axios from "axios";
+
+export function Home() {
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    (async function () {
+      const res = await axios.get("/api/products");
+      const data = await res.data;
+      setProducts(data.products);
+      console.log(products);
```

Comment:
can implement these in try catch block to handle the errors.

Reply:
will fix this